apps/backend/src/schemas/logs.py

```
@@ -126,7 +126,7 @@ class LogAnalytics(BaseModel):
     anomalies_detected: List[Dict[str, Any]] = Field(description="Detected log anomalies")
     trends: Dict[str, List[float]] = Field(description="Log volume trends")
     generated_at: datetime = Field(
-        default_factory=lambda: datetime.now(datetime.UTC), description="Analysis timestamp"
+        default_factory=lambda: datetime.now(timezone.utc), description="Analysis timestamp"
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)